docs/intro-to-eda.md

```diff
@@ -1,56 +1,53 @@
-* The Enterprise Event Bus Team offers consultations and is happy to answer questions. Reach out to us [link to contact page](https://department-of-veterans-affairs.github.io/ves-event-bus-developer-portal/get-support/).
-
-# Having Trouble?
-If you find something wrong with the documentation, didn’t find what you’re looking for, or have a question or suggestion, please [contact us](./get-support.md).
+When determining what information an event should contain, a guiding principle is that consumers need events to at least contain enough information to know whether the event is of interest to them, while minimizing the amount of sensitive data in the event payload. In general it is easier to add new pieces of information to a schema than it is to later remove it; thus the system should trend towards leaner events.
 
+##  Technology overview
+
+Enterprise Event Bus, like most other event-based systems in VA, is based on [Apache Kafka](https://kafka.apache.org/), an open-source distributed event streaming platform. It is considered the industry standard for handling real-time data feeds, and is capable of handling the kinds of large scale, high-throughput loads we would expect an enterprise-wide event bus in VA to be able to handle. In addition to being a proven technology, Kafka is already used in a production capacity by other teams in VA, such as the [Benefits Integration Platform](https://confluence.devops.va.gov/pages/viewpage.action?spaceKey=VAExternal&title=Benefits+Integration+Events).
+
+The Enterprise Event Bus uses [AWS Managed Streams for Kafka (MSK)](https://docs.aws.amazon.com/msk/index.html), a hosted version of Kafka that runs in the VA Enterprise Cloud. 
+
+_The following diagram illustrates how events pass through the different components of the Enterprise Event Bus system. View the diagram in [Github](https://github.com/department-of-veterans-affairs/VES/blob/master/research/Event%20Bus/Diagrams/Data%20Flow.jpeg)._
+![Simple Flow EBus](https://github.com/department-of-veterans-affairs/ves-event-bus-developer-portal/assets/95644573/61c8f134-7228-4735-b9df-c0e1985d9eaa)
 
+## Learn more
+- Find definitions for acronyms and event-related terms on the [Terminology page](./terminology.md).
+- Watch this [brief video](https://www.youtube.com/watch?v=R6tUoxx2gVY) for a quick overview of event-driven architecture.
+- [Learn about event streaming patterns](https://developer.confluent.io/patterns/) on the Confluent Developer website.
+- [Watch a presentation](https://www.confluent.io/events/kafka-summit-americas-2021/improving-veteran-benefit-services-through-efficient-data-streaming/) about “Lessons Learned and Plans for the Future” by Robert Ezekiel from Booz Allen Hamilton.
+- Improving Veteran Benefit Services Through Efficient Data Streaming | Robert Ezekiel, Booz Allen Hamilton Watch a presentation about “Improving Veteran Benefit Services Through Efficient Data Streaming” by Robert Ezekiel from Booz Allen Hamilton
```

Comment:
Let's make these links open in new tabs, since they're external to the portal. Oseas tells me the formatting needs to look like `[google](https://www.google.com/){:target="_blank"}`